Source/ASPagerNode.m

```diff
@@ -115,6 +115,17 @@ - (NSInteger)currentPageIndex
   return (self.view.contentOffset.x / CGRectGetWidth(self.view.bounds));
 }
 
+- (CGSize)pageSize
+{
+  CGSize pageSize = self.bounds.size;
+  UIEdgeInsets contentInset = self.view.contentInset;
+  if (! UIEdgeInsetsEqualToEdgeInsets(UIEdgeInsetsZero, contentInset)) {
+    pageSize.width -= (contentInset.left + contentInset.right);
```

Comment:
Do we want to apply the horizontal inset here? It makes perfect sense to me that users would want to account for content inset in the cross direction, but I'm not so sure about the scrolling direction. For example, if they have a 50pt left inset so that the carousel starts off inset by a bit, they probably still want each page to be full-width, since the insets will only be applied at the start/end of the feed, not on every page. Thoughts?